app/(drawer)/settings.tsx

```diff
@@ -0,0 +1,195 @@
+  const [focusedMenuItem, setFocusedMenuItem] = useState('language'); 
+  const [selectedSubmenuItem, setSelectedSubmenuItem] = useState("English");
+
+  return (
+    <SpatialNavigationRoot isActive={isActive}
+      onDirectionHandledWithoutMovement={onDirectionHandledWithoutMovement}>
+      <Stack.Screen options={{ headerShown: false }} />
+      <Text style={styles.heading}>Account</Text>
+      <View style={styles.container}>
+        
+      <DefaultFocus>
+      <SpatialNavigationFocusableView>
+      <View style={styles.container}>
+       
+      {/* Main Menu (Left) */}
+      <View style={styles.mainMenu}>
+        {menuItems.map((item) => (
+          <TouchableOpacity
+            key={item.id}
+            style={[
+              styles.menuItem,
+              focusedMenuItem === item.id && styles.focusedMenuItem,
+              selectedMenuItem === item.id && styles.selectedMenu,
+            ]}
+            onPress={() => setSelectedMenuItem(item.id)} 
+            onFocus={() => setFocusedMenuItem(item.id)} 
+            onBlur={() => setFocusedMenuItem(null)} 
+          >
+            <Text style={[styles.menuText, selectedMenuItem === item.id && styles.selectedMenuItem]}>{item.label}</Text>
+          </TouchableOpacity>
+        ))}
+      </View>
+
+      {/* Submenu (Right) */}
+     {submenus[selectedMenuItem] && <View style={styles.submenu}>
+      <Text style={styles.title}>Settings</Text>
+        {selectedMenuItem && submenus[selectedMenuItem]?.map((option, index) => (
+          <TouchableOpacity
+            key={index}
+            style={[styles.submenuItem, selectedSubmenuItem === option && styles.focusedSubMenuItem]}
+            onPress={() => console.log(`Selected: ${option}`)} 
+          >
+            <Text style={[styles.submenuText, selectedSubmenuItem === option && styles.selectedSubmenuText]}>{option}</Text>
+          {selectedSubmenuItem === option &&
+              <Icon name="check" size={20} color="#00FF00" style={[styles.checkIcon, selectedSubmenuItem === option && styles.selectedCheckIcon]} />
+          }
+          </TouchableOpacity>
+        ))}
+      </View>
+}
+    </View>
+      </SpatialNavigationFocusableView>
+      </DefaultFocus>
+      </View>
+    </SpatialNavigationRoot>
+  );
+}
+
+const useTVStyles = function() {
+  return StyleSheet.create({
+    container: {
+        flexDirection: 'row',
+        flex: 1,
+        backgroundColor: '#000',
+        justifyContent: 'center',
+        alignItems: 'center',
+      },
+      heading: {
+        fontSize: scaledPixels(32),
+        fontWeight: 'bold',
+        color: '#fff',
+        textAlign: 'center',
+        marginTop: scaledPixels(80),
+      },
+      title: {
+        fontSize: scaledPixels(32),
+        fontWeight: 'bold',
+        color: '#fff',
+        textAlign: 'center',
+        marginBottom: scaledPixels(20),
+      },
+      mainMenu: {
+        width: '40%',
```

Comment:
Can you please share the video for the web, and tv? need to see how this 40% is working.